src/overlays/actors/ovl_En_Mag/z_en_mag.h

```diff
@@ -8,9 +8,51 @@ struct EnMag;
 
 typedef struct EnMag {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0xE1DC];
+    /* 0x014C */ char unk_14C[0x0004];
+    /* 0x0150 */ u8 unk_150[0xE188]; // kanfont struct/buffer
```

Comment:
Yeah there isn't much point in guessing how to break down the struct/array right now. I have no idea how it should be, and it will be done properly with z_message and/or z_kanfont decomp.